eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -126,6 +126,7 @@ if ($relatedTypeSpecProjectFolder) {
           })
         } else {
             $relativeSdkPath = Resolve-Path $sdkProjectFolder -Relative
+            $specRepoRoot = $swaggerDir -replace "/specification", ""
```

Comment:
Why do we need this? in the pipeline, the $swaggerDir does not contain `specification`

Reply:
`swaggerDir` is the folder of specification instead of root folder of spec repo, e.g. `azure-rest-api-specs/specification`.

See [this schema](https://github.com/Azure/azure-sdk-tools/blob/612e3e995ffa4bfd9525d28b44fcb95cded07e54/tools/sdk-generation-pipeline/documents/task-engine/schema/GenerateAndBuildInputSchema.json#L4) for details.